backend/routes/auth_route.py

```diff
@@ -24,6 +24,7 @@ def register():
 
     existing_user = Users.objects(username=data.get('username')).first()
     if existing_user:
+        log_audit(0,request.method, request.path, "Username is already taken", 400, "Authentication Error", str(request.user_agent), request.remote_addr, data.get("username"))
```

Comment:
Maybe actions to be logged should be considered and filtered, storing unnecessary actions is not really feasable